cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -830,7 +834,7 @@ public synchronized void executeProposals(Collection<ExecutionProposal> proposal
                             requestedIntraBrokerPartitionMovementConcurrency, requestedClusterLeadershipMovementConcurrency,
                             requestedBrokerLeadershipMovementConcurrency, requestedExecutionProgressCheckIntervalMs, replicaMovementStrategy,
                             isTriggeredByUserRequest, loadMonitor);
-      startExecution(loadMonitor, null, removedBrokers, replicationThrottle, isTriggeredByUserRequest);
+      startExecution(loadMonitor, null, removedBrokers, replicationThrottle, logDirThrottle, isTriggeredByUserRequest);
```

Comment:
Just curious, does that make more sense to have a separate method for intra-broker throttling since this parameter only applies for inter-broker movement?

Reply:
I agree with you. But, since the `startExecution` method is invoked for all rebalance tasks yet (i.e., both inter-broker and intra-broker with demote, remove broker options), I thought the splitting task is better to be done with independent PR.

It is why I chose to add a new parameter here.

Reply:
Let's create a task to track the split. Thanks!